### ISSUE TITLE
installation fails with being unable to sign up

### ISSUE THREAD

**Issue by marlonn** [appwrite/todo-with-vue#23]
I'm trying to install this app example locally, and I can't get it running. Note that I have limited knowledge of docker, Vue, or appwrite. At first, my todo items weren't saved, Firefox' console said "User id is null/empty". I figured that was because I was unable to login/signup, because the screen redirected immediately to the prompt for entering todo-items, which I was able to mitigate by editing App.vue with this:

```
  watch: {
    getAccount(newValue, oldValue) {
      console.log("In Watch getAccount in App.vue")
      if (newValue === null) {
        this.$router.replace('/')
      } else {
        this.$router.replace('/login')
      }
    }
```

But now I'm stuck trying to sign up with my local instance, see this output of Firefox' javascript console:

```
signing up ... [redacted] SignUp.vue:64:15
XHROPTIONShttps://localhost/account
[HTTP/2 204 No Content 6ms]

XHRPOSThttps://localhost/account
CORS Missing Allow Origin

Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://localhost/account. (Reason: CORS header 'Access-Control-Allow-Origin' missing).

Error creating Account account.js:15:15
XHRGEThttp://localhost:3000/__vite_ping
```

My guess is that I need to do one of these:
- set up the docker network differently (I only just installed docker on this machine, which is running Ubuntu 20.04 (desktop))
- change my answers in the setup script (I kept the defaults, though I made my own API key)
- install this example app in a docker container as well and set it to the same subnet as the appwrite backend (no idea how to do that, I don't have any experience with docker)
- read the instructions more carefully (I doublechecked several times already, but who knows ...)
- use a different operating system than Ubuntu 20.04

tldr: This guide (or the app itself) does not work as-is.

**Comment by eldadfux** [comment]
Seems like you have the wrong endpoint set for Appwrite. `https://localhost/account` should be `https://localhost/v1/account`.

Use `https://localhost/v1` as your endpoint value.

**Comment by marlonn** [comment]
Thanks for taking the time to answer. I got the following Firefox console output with App.vue changed:

```
In Watch getAccount in App.vue App.vue:16:15
signing up ... [redacted] SignUp.vue:64:15
XHRPOSThttp://localhost:3000/localhost/v1/account
[HTTP/1.1 404 Not Found 1ms]

Error creating Account account.js:15:15
```

And this with the original version of App.vue:

```
In Watch getAccount in App.vue App.vue:16:15
XHRGEThttp://localhost:3000/localhost/v1/database/collections/60a875e274122/documents?limit=25&orderType=ASC&orderCast=string
[HTTP/1.1 200 OK 2ms]

GEThttp://localhost:3000/favicon.ico
[HTTP/1.1 200 OK 1ms]

User id is null/empty
```

Seems like localhost/v1 is not right, either - how would I determine which is the right endpoint? Doesn't the appwrite backend display this information anywhere?

**Comment by marlonn** [comment]
Thanks, it works now!

For future reference, how would I figure out myself what the endpoint needs to? I'd assume this would be displayed somewhere in the appwrite console, but where?

**Comment by eldadfux** [comment]
You can view it in your project's settings:

![image](https://user-images.githubusercontent.com/1297371/119260895-8ea0c700-bbdd-11eb-95cd-735b82037c86.png)
